projects/kubernetes-sigs/image-builder/BOTTLEROCKET_RELEASES

```diff
@@ -1,5 +1,5 @@
 1-27:
-  ami-release-version: v1.20.5
+  ami-release-version: v1.22.0
```

Comment:
Bottlerocket 1.27 VMWare variant is no longer supported from v1.21.0 onwards.
PR: https://github.com/bottlerocket-os/bottlerocket/pull/4079

Reply:
Bottlerocket do still have the 1.27 AMI, they dropped only the 1.27 raw and OVA images.